portfolio/src/main/webapp/script.js

```diff
@@ -63,14 +63,23 @@ function previousExperience() {
   selectedExperience--;
 }
 
+var number = 0;
+
+function storeNumberComments() {
```

Comment:
Similarly, I think this function name is a little misleading. From what I can tell, this isn't storing the number of comments, but instead the storing the number of comments to display to the user and then populating the comment section. I'd recommend renaming this too.